browser-test/src/api_docs.test.ts

```diff
@@ -122,16 +129,6 @@ describe('Viewing API docs', () => {
     // causing inconsistent screenshots.
     await page.waitForTimeout(300) // ms
 
-    // Wait for the specific text to become visible in the accordion.
```

Comment:
question (non-blocking): why did you remove this?

Reply:
With the new way of implementing the accordion, there is no longer a `cf-accordion-visible` CSS class and the text in the content div is always present.  What changes is the `display` styling (going from `none` to `block`).  I couldn't figure out how to evaluate whether that styling was present, plus we're already testing that the content is displayed in the screenshot validation.  I can give it another try if you feel that the screenshot validation isn't enough.

Reply:
Gotcha. Thanks for explaining! I think the screenshot is enough since it will change if the text changes.